recognition/include/pcl/recognition/ransac_based/rigid_transform_space.h

```diff
@@ -229,20 +229,20 @@ namespace pcl
           int max_num_transforms = 0;
 
           // For each full leaf
-          for ( std::vector<CellOctree::Node*>::const_iterator leaf = full_leaves.begin () ; leaf != full_leaves.end () ; ++leaf )
+          for (const auto &full_leave : full_leaves)
```

Comment:
Minor: singular from "leaves" is "leaf".

Reply:
Clang automatically only removes the `s` to switch to singular ;).

Reply:
Shame on you, clang!